ai-library/README.md

```diff
@@ -0,0 +1,37 @@
+# AI Library 
+
+The AI Library component installs the AI Library operator as well as the CRD for AI Library.
+
+### Folders
+There are 2 folders that contain installation files:
+
+`cluster/base` This folder includes the yaml files that will install the AI Library CRD on your cluster.
+
+`operator/base` This folder includes the yaml files to set up the deployment for the operator in your namespace along with the service account, role, and role binding that is necessary.
+
+### Installation
+To install AI Library add the following to the `kfctl` yaml file.
+
+Note, to use AI Library, you also need to have Seldon installed.
```

Comment:
Should this have a link to Seldon README and maybe mention that Seldon has to come before AI Library?

Reply:
Happy to add that.